Sources/Fullscreen/TransactionView/TransactionStepView/TransactionStepView.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol TransactionStepViewDelegate: AnyObject {
+    func transactionStepViewDidSelectActionButton(_ view: TransactionStepView, inTransactionStep step: Int)
+}
+
+public class TransactionStepView: UIView {
+    // MARK: - Public properties
+
+    public weak var delegate: TransactionStepViewDelegate?
+
+    // MARK: - Private properties
+
+    private var step: Int
+    private var model: TransactionStepViewModel
+    private var style: TransactionStepView.Style
+
+    private lazy var verticalStackView: UIStackView = {
+        let stackView = UIStackView(withAutoLayout: true)
+        stackView.axis = .vertical
+        stackView.distribution = .fillProportionally
+        stackView.alignment = .leading
+        return stackView
+    }()
+
+    private lazy var titleView: UITextView = {
+        let view = UITextView(withAutoLayout: true)
+        view.font = style.titleFont
+        view.textColor = style.titleTextColor
+        view.backgroundColor = style.backgroundColor
+        view.isScrollEnabled = false
+        view.isEditable = false
+        view.contentInset = .init(top: -.mediumSpacing, leading: 0, bottom: 0, trailing: 0)
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
is `view.translatesAutoresizingMaskIntoConstraints = false` needed if you are using `UITextView(withAutoLayout: true)`

Reply:
Will fix 👍